static/app/components/sentryDocumentTitle.tsx

```diff
@@ -1,7 +1,7 @@
 import {createContext, useContext, useEffect, useMemo} from 'react';
 
 type Props = {
-  children?: React.ReactChild;
+  children?: React.ReactNode;
```

Comment:
The difference here is whether one child is allowed, or more than one.

**Before:**
```
<SentryDocumentTitle>
  <SingleChildOnly />
</SentryDocumentTitle>
```

**After:**
```
<SentryDocumentTitle>
  <ChildOne />
  <ChildTwo />
</SentryDocumentTitle>
```